docs/dev/log_integration.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Create a Log Integration
+kind: documentation
+aliases:
+- /logs/faq/partner_log_integration
```

Comment:
For some reason, this alias isn't working for the preview, not sure if that's expected.

Reply:
I just tried it on the [preview link](https://docs-staging.datadoghq.com/alai97/log-integration-guide-to-page/logs/faq/partner_log_integration/) and it's working for me!